documentation/b2b.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: master
+permalink: /documentation/b2b/
+title: B2B
+weight: 7
+---
+# B2B Storefront Configuration and Implementation
+
+## Configuration
+TBA
+
+## Implementing KeyCloak Theme
+By default, KeyCloak provides a set of themes, such as, login or accounts, that can be used for the various actions that KeyCloak supports. Use one of these existing themes for the supported flows or add a customized theme to the KeyCloak Docker Image, if required. The REACT PWA Reference Storefront provides a login theme to use for the custom login page.
```

Comment:
Do you want to add a link to the Deployment guide here
Keycloak Docker image part is addressed in that doc :)

Reply:
Sure! Do you have a link to the Account Management deployment guide? :)